modules/modules.mk

```diff
@@ -59,7 +64,7 @@ ifeq ($(PHASE_FIELD),yes)
 endif
 
 # The master list of all moose modules
-MODULE_NAMES := "chemical_reactions contact fluid_properties functional_expansion_tools heat_conduction level_set misc navier_stokes phase_field porous_flow rdg richards solid_mechanics stochastic_tools tensor_mechanics xfem external_petsc_solver"
+MODULE_NAMES := "chemical_reactions contact fluid_properties functional_expansion_tools heat_conduction level_set misc navier_stokes peridynamics phase_field porous_flow rdg richards solid_mechanics stochastic_tools tensor_mechanics xfem"
```

Comment:
Oh no! It looks like another module was added to the list.  We're going to have to fix that (and restart all the testing).

Reply:
We need to modify this file to add peridynamics module into the list, right?

Reply:
Yes.  All you need to do is add 'external_petsc_solver' to the end of the list. Actually, add it where it is supposed to be after 'contact' so that they're all in alphabetical order.

Reply:
Actually, I just edited it myself, so don't worry about it.  Hopefully the tests all pass again!